src/components/applications/application.tsx

```diff
@@ -26,16 +28,10 @@ const Application = ({ title, questions }: props) => {
         }}
       />
 
-      <Text
-        meta={{
-          type: "text",
-          title: "Email",
-          placeholder: "Enter an Email",
-          maxLength: 250,
-          value: "SAMPLE EMAIL",
-          disabled: true,
-        }}
-      />
+      <div>
+        <Label htmlFor="email">Email</Label>
```

Comment:
should be in its own component.

you can move this code to /components/global/inputs/input.tsx

Reply:
Wait so I'm a bit confused on what I should do. Should I make a seperate component that contains both a label and input field?

Reply:
yeah there should be a separate component containing both label and input.

but you don't need to create a new file for it, there's already a file to put your code in.

Reply:
https://github.com/acm-ucr/starlight/blob/dev/src/components/global/inputs/input.tsx 
in here